### DISCUSSION TITLE
Add support for partial properties from .NET 9

### DISCUSSION THREAD

**Discussion by dominikjeske** [canton7/PropertyChanged.SourceGenerator#46] [Ideas]
We can now use partial props https://learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-9/overview#c-13 so maybe add support for this in PropertyChanged.SourceGenerator?

**Comment by dominikjeske** [comment]
duplicate

**Reply by canton7** [reply]
It's ongoing, as with everything in life it's about finding the time...
